unittest/Dialect/TritonGPU/LinearLayoutConversionsTest.cpp

```diff
@@ -311,6 +311,58 @@ TEST_F(LinearLayoutConversionsTest, Blocked4D) {
                     {S("dim0"), S("dim1"), S("dim2"), S("dim3")}));
 }
 
+TEST_F(LinearLayoutConversionsTest, BlockedDotOperandLhs) {
```

Comment:
@lezcano 
I want to add dot3d operand tests as well. if you need this functionality ASAP, I will add them in a followup PR.
I've tested this PR with pytest test and everything seems good so far.